### PR TITLE
Add responses to rails api diagnostic

### PR DESCRIPTION
**Pull request by jim-moody**
Comfort: 5
Clarity: 5

I used the following documentation for help with these answers:
https://github.com/jim-moody/rails-api-single-resource
https://github.com/rails-api/rails-api
https://github.com/ga-wdi-boston/rails-api-library-demo

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -5,74 +5,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+To retrieve and save data and do data manipulation
```
Comment (jordanallain) [inline diagnostic.md:8]: Also to allow multiple users to interact on the same app.

File: diagnostic.md [hunk 2]
```diff
@@ -5,74 +5,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
...
+* The controller must have a `show` method which takes in the `id` param and calls the `Person` model
+* The `Person` model must be created and call out to the database for a `Person`
+* The database must have a people table with a person of id=1
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+rails-api new api_name
```
Comment (jordanallain) [inline diagnostic.md:67]: Good I'm assuming you meant `rails-api new app_name` and not api_name?

File: diagnostic.md [hunk 3]
```diff
@@ -5,74 +5,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
...
 
 What are the commands to drop, create, migrate and seed a database from the command
 line? (5 bullet points)
 
 ```bash
-// your response here
+* db:drop
+* db:create
+* db:migrate
+* db:seed
```
Comment (jordanallain) [inline diagnostic.md:83]: Last one would be db:examples

File: diagnostic.md [hunk 4]
```diff
@@ -5,74 +5,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
...
 ```
 
 What is the command to scaffold a pet with a name and age attributes (hint:
 Also think of the data types for each attribute)?
 
 ```bash
-// your response here
+rails generate model Pet name:string age:integer
```
Comment (jordanallain) [inline diagnostic.md:90]: Good and we don't need to capitalize pet here.

**Comment by jordanallain** [comment]
12/12

💯 Moody on 🔥 in the streets! 🚒 

🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴 🇯🇴

File: diagnostic.md (hunk 2 above)
Comment (jim-moody) [reply diagnostic.md:67]: Yes :)

**Pull request closed without merging**